linode/manager
# upcoming: [DI-21322] - Retain Resource selections during expand / collapse of filter button

## Description 📝
Bug fix for retaining the resource selections during expand / collapse of filter button

## Changes  🔄

1.Bug fix for retaining the resource selections during expand / collapse of filter button 

## Target release date 🗓️
10-10-2024

## Preview 📷
**Include a screenshot or screen recording of the change**

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
|![Screenshot 2024-10-14 at 8 01 38 PM](https://github.com/user-attachments/assets/c6b067cf-8703-4711-8095-6051b92b8c9e)|![Screenshot 2024-10-14 at 8 00 41 PM](https://github.com/user-attachments/assets/3665d9a2-03d4-4f7d-b0f6-182d0be27388)|

## How to test 🧪

1. Login as mock user
2. Navigate to monitor tab
3. Select a dashboard and in filters section select upto resources / cluster
4. Collapse and expand the filter button, the selected resources should be retained

### Reproduction steps

Clone the linode/manager/develop branch and do the following steps

1. Login as mock user
2. Navigate to monitor tab
3. Select a dashboard and in filters section select upto resources / cluster
4. Collapse and expand the filter button, **the selected resources will not be retained**

## As an Author I have considered 🤔

*Check all that apply*
